lib/spack/spack/util/executable.py

```diff
@@ -146,7 +146,7 @@ def streamify(arg, mode):
 
         quoted_args = [arg for arg in args if re.search(r'^"|^\'|"$|\'$', arg)]
         if quoted_args:
-            tty.warn(
+            tty.debug(
```

Comment:
Are there times this is just a nuisance and not a real bug?  I think it's actually usually worrisome to see this.

Reply:
I thought this was coming up repeatedly on non-problematic code, but I must have edited that away. I'll revert this.